common/src/Microsoft.Azure.IIoT.Agent.Framework/src/Storage/Database/Services/JobDatabase.cs

```diff
@@ -85,26 +103,31 @@ public class JobDatabase : IJobRepository {
                     var result = await _documents.ReplaceAsync(document, updated, ct);
                     return result.Value.ToFrameworkModel();
                 }
-                catch (ResourceOutOfDateException) {
+                catch (ResourceOutOfDateException ex) {
+                    exceptions.Add(ex);
```

Comment:
Should we also add a log here ?

Reply:
Good point, I brushed up a bit the logs in other places as well.